index.html

```diff
@@ -1044,48 +1060,101 @@
-                    the <a for=
-                    "PresentationConnectionAvailableEvent">connection</a>
-                    attribute initialized to <var>S</var>, at
-                    <var>presentationRequest</var>. The event must not bubble,
-                    must not be cancelable, and has no default action.
-                  </li>
-                  <li>
-                    <a>Establish a presentation connection</a> with
-                    <var>S</var>.
-                  </li>
-                  <li>Abort all remaining steps.
-                  </li>
-                </ol>
-              </div>
+            <li>Search the <a>set of controlling presentations</a> for a
+            <a>PresentationConnection</a> that meets the following criteria:
+            its <a>controlling browsing context</a> is the current <a>browsing
+            context</a>, its <a>presentation connection state</a> is not
+              <a for="PresentationConnectionState">terminated</a>, its
+              <a>presentation URL</a> is equal to the <a>presentation request
+              URL</a> of <var>presentationRequest</var>, and its
+              <a>presentation identifier</a> is equal to
+              <var>presentationId</var>.
+            </li>
+            <li>If such a <a>PresentationConnection</a> exists, run the
+            following steps:
+              <ol>
+                <li>Let <var>S</var> be that <a>PresentationConnection</a>.
+                </li>
+                <li>
+                  <a>Resolve</a> <var>P</var> with <var>S</var>.
+                </li>
+                <li>
+                  <a>Queue a task</a> to <a>fire</a> a <a>trusted event</a>
+                  with the name <a>connectionavailable</a>, that uses the
+                  <a>PresentationConnectionAvailableEvent</a> interface, with
+                  the <a for=
+                  "PresentationConnectionAvailableEvent">connection</a>
+                  attribute initialized to <var>S</var>, at
+                  <var>presentationRequest</var>. The event must not bubble,
+                  must not be cancelable, and has no default action.
+                </li>
+                <li>If the <a>presentation connection state</a> of <var>S</var>
+                is <a for="PresentationConnectionState">connecting</a> or
+                <a for="PresentationConnectionState">connected</a>, then abort
+                all remaining steps.
+                </li>
+                <li>Set the <a>presentation connection state</a> of
+                <var>S</var> to <a for=
+                "PresentationConnectionState">connecting</a>.
+                </li>
+                <li>
+                  <a>Establish a presentation connection</a> with <var>S</var>.
+                </li>
+                <li>Abort all remaining steps.
+                </li>
+              </ol>
+            </li>
+            <li>Search the <a>set of controlling presentations</a> for the
```

Comment:
These steps are similar to the one above and I thought they were identical... Oops.